packages/@aws-cdk-testing/framework-integ/test/aws-lambda-nodejs/test/integ.esbuildArgs.js.snapshot/tree.json

```diff
@@ -20,8 +20,8 @@
                     "id": "ImportServiceRole",
                     "path": "cdk-integ-lambda-nodejs-esbuildArgs/ts-handler/ServiceRole/ImportServiceRole",
                     "constructInfo": {
-                      "fqn": "constructs.Construct",
-                      "version": "10.3.0"
+                      "fqn": "aws-cdk-lib.Resource",
+                      "version": "0.0.0"
```

Comment:
Are these supposed to be changing like this? Are these updates that should have happened but are only being found now?

Reply:
Unrelated, but these haven't been updated in a while so it's likely as a result of the jsii changes that were merged recently. I expected some of this.